tests/NovelRT.Tests/CMakeLists.txt

```diff
@@ -56,18 +56,19 @@ set(SOURCES
 add_executable(Engine_Tests ${SOURCES})
 
 if(NOVELRT_BUILD_INTEROP)
-    copy_build_products(Engine_Tests
-      DEPENDENCY Engine
-      TARGET_LOCATION $<TARGET_FILE_DIR:Engine_Tests>
+  copy_build_products(Engine_Tests
+    DEPENDENCY Engine
+    TARGET_LOCATION $<TARGET_FILE_DIR:Engine_Tests>
```

Comment:
What actually changed here? Just spacing?

Reply:
Yep, just made spacing more consistent as for some reason certain IDEs seem to indent farther than they should with CMakeLists